sources/editor/Stride.GameStudio/Stride.GameStudio.csproj

```diff
@@ -34,9 +34,9 @@
     <PackageReference Include="SharpDX" Version="4.2.0" />
     <PackageReference Include="Stride.Metrics" Version="1.0.3" />
     <PackageReference Include="Stride.CrashReport" Version="1.0.2" />
-    <PackageReference Include="Mono.Cecil" Version="0.11.2" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.5" />
     <Reference Include="Stride.Core.AssemblyProcessor">
-      <HintPath Condition="$(TargetFramework.StartsWith('net6'))">..\..\..\deps\AssemblyProcessor\netstandard2.0\Stride.Core.AssemblyProcessor.dll</HintPath>
+      <HintPath>..\..\..\deps\AssemblyProcessor\$(StrideFramework)\Stride.Core.AssemblyProcessor.dll</HintPath>
```

Comment:
If you're referring to `$(StrideFramework)` here would it make sense to use it in the AssemblyProcessor project as well (after importing the props that defines it)?

Reply:
Yes, maybe that would be a problem. As it is a direct reference to the DLL, it becomes locked when building Game Studio. Or so I think. I remember having read something about avoiding a file lock when building projects in parallel or something.

Maybe if that is replaced with a project reference it'll work?
A project reference would in theory copy the output DLL to the output directory of GameStudio and load it from there, isn't it?
But, about that DLL, the output of Assembly Processor is not just that DLL, it's a different one resulting from packing the Assembly Processor with its dependencies.

Aside: What's the actual advantage of packing the Assembly Processor with its dependencies?

Reply:
I think that may have been a variant of a SingleFilePublish before that was a thing to get a minimal executable to be invoked.
Would be good to understand what is the dependency of the editor on the assembly processor.

Reply:
A little idea I had today. Would there be any problem if GameStudio consumes the AssemblyProcessor as a project reference?

That way, the DLL in `deps/AssemblyProcessor` is only used by the SDK to copy to `%TEMP%` and load the MSBuild task from there, as it does right now. But GameStudio, that uses the AssemblyProcessor directly, would consume the DLL from the project output, MSBuild manages the copy of newer versions, etc., and (with luck) there will be no locks while building the solution.

May it work?

Reply:
Sorry for the delay.
Good idea, I think this approach might work.
One point to check is how it behaves when GameStudio runs a build (it is running its own MSBuild instance), but I think it should be OK.